packages/scaffold-ui/src/partials/w3m-account-default-widget/index.ts

```diff
@@ -87,9 +88,12 @@ export class W3mAccountDefaultWidget extends LitElement {
         gap="l"
       >
         <wui-profile-button-v2
-          .onProfileClick=${this.handleSwitchAccountsView.bind(this)}
+          .onProfileClick=${ChainController.state.activeChain !== 'solana' &&
+          this.handleSwitchAccountsView.bind(this)}
           address=${ifDefined(this.address)}
-          icon="${account?.type === 'smartAccount' ? 'lightbulb' : 'mail'}"
+          icon="${account?.type === 'smartAccount' && ChainController.state.activeChain !== 'solana'
```

Comment:
We should use `W3mFrameRpcConstants.ACCOUNT_TYPES.SMART_ACCOUNT` from `@web3modal/wallet` to not use hard coded strings